app/models.py

```diff
@@ -140,6 +140,17 @@ class User(db.Model):
     def password(self):
         raise AttributeError("Password not readable")
 
+    @property
+    def can_use_webauthn(self):
+        if self.platform_admin:
+            return True
+
+        return any([
+            str(service.organisation_id) == current_app.config['BROADCAST_ORGANISATION_ID'] or
+            str(service.id) == current_app.config['NOTIFY_SERVICE_ID']
```

Comment:
Who is this check for? What's the flow? have i understood this right?

* new user joins notify team
* A new manual step: an existing platform admin invites them to the notify service
* they can now use webauthn, so set up their security key
* we enable platform admin in the db after verifying they're set up to use webauthn_auth
* They can leave the notify service if they want

*repeat for all three environments*

if we're starting to automate this flow, is this the process we want to use? We still need to go into the database manually to set someone to platform_admin, so it's not a fully automated process (although this does remove one manual prod database write access, which is nice). I guess we won't use this process too much anyway so I'm not too worried.

Reply:
I mentioned this in the 2nd commit:

> Allowing members of the GOV.UK Notify service to use the feature
> is a workaround, so we can avoid making someone a Platform Admin
> before they've protected their account with [WebAuthn].

> if we're starting to automate this flow, is this the process we want to use?

No, definitely not 😄. However, I don't think we're going to get to a full roll out of WebAuthn for a long time. This isn't so much about making the process easier, as making it more secure (see above).

I've added a task to the card to remind me to update the guidance for new Platform Admins, if this goes in.